kpatch/kpatch

```diff
@@ -426,6 +429,18 @@ remove_module () {
 	rmmod "$modname" 2> /dev/null || return 0
 }
 
+remove_module_quiet () {
```

Comment:
Looks good. Silencing the echo is a good idea but it is hardly worth a full copy of remove_module_quiet() function. We can add an optional variable instead and only set it to "true" or something in the new call, leaving old ones as is.